.pre-commit-config.yaml

```diff
@@ -83,6 +83,12 @@ repos:
             language: system
             pass_filenames: false
 
+          - id: go_integration
```

Comment:
Do not run integration tests in pre-commit. This requires credentials, which some devs might not have set up (like me)

Reply:
modified